packages/terra-abstract-modal/CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ Changelog
 Unreleased
 ----------
 ### Changed
-Updated `keycode-js` from `v1.0.4` to `v2.0.1`
+* Cleaned up imports in examples and test files
+* Updated `keycode-js` from `v1.0.4` to `v2.0.1`
```

Comment:
Is this part of your change ?

Reply:
It was a merge conflict in some changelogs